R/apply_filters.R

```diff
@@ -71,19 +158,34 @@ apply_filters <- function(data,
   remove <- data %>%
     dplyr::filter(
       .data$plan_tech_prod == 0,
-      .data$year == .env$start_analysis
-    ) %>%
-    dplyr::mutate(
-      key = paste(.data$company_name, .data$id, .data$ald_sector, .data$technology, .data$scenario_geography)
+      .data$year == .env$start_analysis,
+      .data$technology %in% .env$declining_technologies
```

Comment:
Just to go sure, the difference between apply_filters and apply_filters old is that in apply filters all decling technologies are removed?

Reply:
if yes maybe also add this to the description to disambiguiate it from apply_fiters_olds

Reply:
it is about identifying which company/technology combination has zero production value in the start year and removing that. Given the recent changes, this is not necessary anymore and should be removed altogether

Reply:
apply filters was removed